etc/logging.json

```diff
@@ -42,7 +42,7 @@
     },
     "root": {
         "handlers": ["default", "file"],
-        "level": "WARNING"
+        "level": "INFO"
```

Comment:
Think we probably want to keep the default log level in our logging config to `WARNING`. @hansmosh or @f1401martin can you confirm we don't want to change this?